x-pack/plugins/security_solution/public/timelines/components/flyout/header/translations.ts

```diff
@@ -77,3 +81,17 @@ export const TIMELINE_TOGGLE_BUTTON_ARIA_LABEL = ({
     values: { isOpen, title },
     defaultMessage: '{isOpen, select, false {Open} true {Close} other {Toggle}} timeline {title}',
   });
+
+export const SAVE_TOUR_CLOSE = i18n.translate(
+  'xpack.securitySolution.timeline.flyout.saveTour.closeButton',
+  {
+    defaultMessage: 'Close',
+  }
+);
+
+export const SAVE_TOUR_TITLE = i18n.translate(
+  'xpack.securitySolution.timeline.flyout.saveTour.title',
+  {
+    defaultMessage: 'Timelines are now saved manually',
```

Comment:
@janmonschke are we saying that users have to manually save Timelines now? If so, I'd change this text to the following:

```suggestion
    defaultMessage: 'Timelines now require manual saves',
```